Mathlib/Data/Real/EReal.lean

```diff
@@ -900,6 +900,24 @@ theorem add_lt_top {x y : EReal} (hx : x ≠ ⊤) (hy : y ≠ ⊤) : x + y < ⊤
   exact EReal.add_lt_add hx.lt_top hy.lt_top
 #align ereal.add_lt_top EReal.add_lt_top
 
+theorem add_le_add_right {x y z : EReal} (h : x ≤ y) : x + z ≤ y + z := by
+  rcases lt_or_eq_of_le h with (h | rfl)
+  swap; exact le_rfl
+  induction z with
+  | h_bot => simp
+  | h_real z => exact add_lt_add_right_coe h z |>.le
+  | h_top => exact (add_top_of_ne_bot h.ne_bot).symm ▸ le_top
+
+theorem add_le_add_left {x y z : EReal} (h : x ≤ y) : z + x ≤ z + y :=
```

Comment:
All the theorems in your PR should be lemmas. The rule (not yet enforced) is that `theorem` is only for important statements, and in particular all theorems should have a docstring.